x-pack/plugins/ml/public/jobs/jobs_list_new/components/jobs_list/styles/main.less

```diff
@@ -0,0 +1,106 @@
+.jobs-list-table > div > .euiTable {
+  & > thead {
+    th {
+      text-align: left;
+      white-space: normal;
+    }
+    th:nth-child(2) {
```

Comment:
Might be worth adding some css magic to resize these when the total width shrinks to a certain limit

Reply:
added as a task to https://github.com/elastic/kibana/issues/20150